### PR TITLE
feat: Send MLS messages (FS-560)

### PR DESCRIPTION
**Pull request by thisisamir98**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/FS-560" title="FS-560" target="_blank">TaskFS-560</a>  [Web] Send message in an MLS group
  </td></table>
  <br />
 

send mls messages

### CODE REVIEW

File: src/script/conversation/ConversationRepository.ts [hunk 1]
```diff
@@ -396,7 +397,7 @@ export class ConversationRepository {
      * we need to add this creator_client to conversation creation payload
      * for creating MLS conversations
      */
-    if (this.teamState.isMLSEnabled()) {
```
Comment (tlebon) [inline src/script/conversation/ConversationRepository.ts:399]: do we still need to make sure MLS is enabled in the team features?

Comment (thisisamir98) [reply src/script/conversation/ConversationRepository.ts:399]: image
this is already changed, now we're checking protocol from the options instead

Comment (tlebon) [reply src/script/conversation/ConversationRepository.ts:399]: im asking if *also* need to check that its enabled for the team

Comment (atomrc) [reply src/script/conversation/ConversationRepository.ts:399]: It's checked at another level. At this point, if the conversation is marked as MLS, we can trust that the team config is correct and we need to handle it as such :)

**Review by arjita-mitra — APPROVED** [review]

**Review by tlebon — APPROVED** [review]

**Pull request merged**